modules/lb_membership_calculator/config/optional/field.storage.block_content.field_block.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - block_content
```

Comment:
add it to module dependencies, please https://github.com/YCloudYUSA/yusaopeny_memberships/pull/6

Reply:
@podarok this module  is already in the dependencies list